dev/docker/trino/Dockerfile

```diff
@@ -67,7 +67,7 @@ RUN mkdir /tmp/gravitino
 COPY --chown=trino:trino packages/gravitino-trino-connector /tmp/gravitino
 
 ARG IMAGE_NAME
-RUN if [ "$IMAGE_NAME" = "datastrato/trino" ] ; then \
+RUN if [ "$IMAGE_NAME" = "apache/gravitino-*:trino-*" ] ; then \
```

Comment:
@diqiu50 
Please help to verify this part.

Reply:
I this this is better:
```
apache/gravitino-trino-*:*
```

Reply:
After this PR, the name for Trino Docker image will be:

apache/gravitino-ci:trino-xxxx
apache/gravitino-playground:trino-xxxx

Reply:
@diqiu50 
We will use the tag name to indicate whether it's a Trino image, so please help verify its accuracy.

Reply:
That's ok.